tensorflow_datasets/scripts/document_datasets_test.py

```diff
@@ -40,20 +40,20 @@ class DummyMnistConfigs(DummyMnist):
 
 
 class DocumentDatasetsTest(testing.TestCase):
-
+  """Test for document_datasets module"""
   @classmethod
-  def setUpClass(cls):
+  def setUpClass(cls): # pylint: disable = invalid-name
     super(DocumentDatasetsTest, cls).setUpClass()
     cls._tfds_tmp_dir = testing.make_tmp_dir()
     builder = DummyMnist(data_dir=cls._tfds_tmp_dir)
     builder.download_and_prepare()
 
   @classmethod
-  def tearDownClass(cls):
+  def tearDownClass(cls): # pylint: disable = invalid-name
```

Comment:
Same as above

Reply:
We can change this.
Thank you for pointing this out.